mst/dblink-extension.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Using the `dblink` extension in Managed Service for TimescaleDB
+excerpt: Learn how to use `dblink` extension and connect to other PostgreSQL databases
+tags: [extension]
+---
+
+# Use the PostgreSQL `dblink` extension
```

Comment:
I think we can probably take the backticks out of headings. it's more likely to break something than do anything useful.

Reply:
done!!